### PR TITLE
Clearer format when logging with rosconsole. Some other nits modified

### PR DESCRIPTION
**Pull request by ivanpauno**
Logging style wasn't clear, this solves the problem. Also some codding style modifications from #53.
Concurrent access protection commented on https://github.com/Intermodalics/ros_android/pull/53#discussion_r240840549 hasn't been checked yet.

Partially solves #70

### CODE REVIEW

File: patches/rosconsole.patch [hunk 1]
```diff
@@ -78,37 +78,34 @@
...
++    default:
++      return;
 +  }
 +
 +  std::string name = getName(handle);
 +
 +  std::stringstream ss;
-+  ss << name << ": in file:" << file << ":" << line;
-+  __android_log_print(android_log_level, ss.str().c_str(), "%s", str);
++  ss << ": " << str;
```
Comment (meyerj) [inline patches/rosconsole.patch]: Why is `stringstream` and the additional colon needed here?
What about
```cpp
__android_log_print(android_log_level, name.c_str(), "%s", str.c_str());
```
?

Comment (ivanpauno) [reply patches/rosconsole.patch]: I though that the message appeared juxtaposed to the tag, but that doesn't happen. I will correct it in a moment.

Comment (ivanpauno) [reply patches/rosconsole.patch]: Corrected. 

Example output:
`01-07 13:46:04.408 30065 30080 I ros.hello_ros: nothing_important`
Example output of commented version with file name and line number:
`01-07 14:17:56.980  6073  6090 I ros.hello_ros: in file '/opt/ros_android/example_workspace/src/hello_world_example_app/app/src/main/cpp/main.cpp', line '40': nothing_important`

**Review by jubeira — APPROVED** [review]
Output looks good here; I don't have any other comments regarding this so LGTM.

**Review by meyerj — APPROVED** [review]

**Pull request merged**